pkg/ovs/openflow/logs.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/sirupsen/logrus"
```

Comment:
Could you organize imports in groups with blank line between them: standard packages, then 3rd packets?

Reply:
Thank you for your comment, I'll do as you suggested.